backend/.vscode/settings.json

```diff
@@ -7,7 +7,7 @@
     "[python]": {
         "editor.formatOnPaste": false,
         "editor.codeActionsOnSave": {
-            "source.fixAll": true
+            "source.fixAll": "explicit"
```

Comment:
Looks like VSCode transitioned to using enums in the newer release. We might want to keep this as a boolean for now, until support is completely removed. 

https://stackoverflow.com/questions/77637621/vscode-workspace-settings-change-on-its-own